lerna.json

```diff
@@ -1,5 +1,13 @@
 {
-  "packages": ["apps/*", "packages/*", "scripts", "packages/fluentui/*", "tools", "typings"],
+  "packages": [
+    "apps/*",
+    "packages/*",
+    "packages/react-components/*",
```

Comment:
Added new packages/react-components directory so apps like the `ts-minbar-test` dont break